test/presenters/return-logs/download-return-log.presenter.test.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const ReturnLogModel = require('../../../app/models/return-log.model.js')
+const ReturnSubmissionModel = require('../../../app/models/return-submission.model.js')
+const ReturnSubmissionLineModel = require('../../../app/models/return-submission-line.model.js')
+
+// Thing under test
+const DownloadReturnLogPresenter = require('../../../app/presenters/return-logs/download-return-log.presenter.js')
+
+describe('Download Return Log presenter', () => {
+  let returnLog
+
+  beforeEach(() => {
+    returnLog = _testReturnLog()
+  })
+
+  describe('the "data" property', () => {
+    describe('when provided with a returnLog with a returnSubmission.method of "abstractionVolumes"', () => {
+      it('correctly formats the data to a csv string', () => {
```

Comment:
The `it()` blocks don't actually tell us what the presenter is doing in each of these scenarios. It would be better if it gave a brief explanation of what the presenter does in each case.